File: abc/307/b.cpp
#include <iostream>
#include <vector>
#include <string>
#include <algorithm>

using namespace std;

int main() {
    int n;
    cin >> n;
    vector<string> strings(n);

    for(int i = 0; i < n; i++)
        cin >> strings[i];

    bool found = false;

    for(int i = 0; i < n; i++) {
        for(int j = 0; j < n; j++) {
            if(i != j) {
                string word = strings[i] + strings[j];
                string reversed_word = word;
                reverse(reversed_word.begin(), reversed_word.end());
                if(word == reversed_word) {
                    found = true;
                    break;
                }
            }
        }
        if(found)
            break;
    }

    if(found)
        cout << "Yes" << endl;
    else
        cout << "No" << endl;

    return 0;
}

File: abc/307/b.py
n = int(input())
strings = list(map(str, input().split()))
found = False

for i in range(0, n):
    for j in range(0, n):
        if i != j:
            word = strings[i] + strings[j]
            if word == word[::-1]:
                found = True
                break
    if found:
        break

if found:
    print("Yes")
else:
    print("No")
